storage/client/client.go

```diff
@@ -1233,16 +1233,30 @@ func (c *StorageClient) RuntimeTransactions(ctx context.Context, p apiTypes.GetR
 			encryptionEnvelope.Format = *encryptionEnvelopeFormat
 			t.EncryptionEnvelope = &encryptionEnvelope
 		}
-		// TODO: Here we render Ethereum-compatible address preimages. That's
-		// a little odd to do in the database layer. Move this farther out if
-		// we have the energy.
+
+		// Render Ethereum-compatible address preimages.
+		// TODO: That's a little odd to do in the database layer. Move this farther
+		// out if we have the energy.
 		if sender0PreimageContextIdentifier != nil && sender0PreimageContextVersion != nil {
 			t.Sender0Eth = EthChecksumAddrFromPreimage(*sender0PreimageContextIdentifier, *sender0PreimageContextVersion, sender0PreimageData)
 		}
 		if toPreimageContextIdentifier != nil && toPreimageContextVersion != nil {
 			t.ToEth = EthChecksumAddrFromPreimage(*toPreimageContextIdentifier, *toPreimageContextVersion, toPreimageData)
 		}
 
+		// Heuristically decide if this is a native runtime token transfer.
+		// TODO: Similarly to above, this application logic doesn't belong here (= the DB layer);
+		// move it out if we establish a separate app/logic layer.
+		if t.Method != nil {
+			if *t.Method == "accounts.Transfer" {
```

Comment:
put this on the list of places to think about if a paratime ever uses multiple denominations in the accounts system

Reply:
Yeah :/, I thought about it.
The alternative is to add the requirement that `body.amount.Denomination == ""`, where `body` is the unstructured JSON column in the DB, and that's a little brittle in its own right. So I just went with the less-code option.